### PR TITLE
translate es project, installation/index.md page

### PR DESCRIPTION
**Pull request by alonsolopezr**
Signed-off-by: Alonso López Romo <alonsolopez@uthermosillo.edu.mx>

**Description**

This PR fixes #

**Notes for Reviewers**
@jesuslerma Tag for review... gracias por el apoyo

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/es/project/installation/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Para actualizar `mesheryctl`, ejecuta el siguiente comando:
+
+```bash
+scoop update mesheryctl
+```
+
+# Instalación Advanzada
+
+Los Userios pueden controlar la imagen de contenedor específico y etiqueta(versión) de Meshery que quieran ejecutar mediante la edición de su archivo local `~/.meshery/meshery.yaml` (un archivo de docker compose).
```
Comment (jesuslerma) [inline docs/pages/es/project/installation/index.md:94]: Los usuarios*

File: docs/pages/es/project/installation/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+services:
+  meshery:
+    image: layer5/meshery:v0.5.0
+    labels:
+      - "com.centurylinklabs.watchtower.enable=true"
+```
+
+---
+
+Cuando Meshery esta levantado y corriendo, las instrucciones para acceder Meshery serán impresas en la pantalla y su navegador web por default deberá ser dirigido a la pantalla de ingreso de Meshery.
```
Comment (jesuslerma) [inline docs/pages/es/project/installation/index.md:108]: está levantado y corriendo.

File: docs/pages/es/project/installation/index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+To upgrade `mesheryctl`, execute the following command:
+
+```bash
+brew upgrade mesheryctl
+```
+
+#### Bash
+
+**Installing with Bash**
```
Comment (jesuslerma) [inline docs/pages/es/project/installation/index.md:45]: Instalación con Bash

File: docs/pages/es/project/installation/index.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+Install `mesheryctl` and run Meshery on Mac or Linux with this script:
```
Comment (jesuslerma) [inline docs/pages/es/project/installation/index.md:47]: Instala `mesheryctl` y ejecuta Meshery en Mac o Linux con este script:

**Review by jesuslerma — CHANGES_REQUESTED** [review]
Hello @alonsolopezr I just added some changes on spelling and also there is a missing part that requires to be translated: Installing with Bash :)

**Comment by leecalcote** [comment]
It's good to see this, @alonsolopezr. 👍

**Pull request closed without merging**